modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableColumnHeader.java

```diff
@@ -586,8 +587,177 @@ private void doColumnAutoSize(TableColumnBase<?,?> column, int cellsToMeasure) {
 
         // if the prefWidth has been set, we do _not_ autosize columns
         if (prefWidth == DEFAULT_COLUMN_WIDTH) {
-            TableSkinUtils.resizeColumnToFitContent(getTableSkin(), column, cellsToMeasure);
-//            getTableViewSkin().resizeColumnToFitContent(column, cellsToMeasure);
+//            TableSkinUtils.resizeColumnToFitContent(getTableSkin(), column, cellsToMeasure);
+            resizeColumnToFitContent(getTableSkin(), column, cellsToMeasure);
+        }
+    }
+
+    /**
+     * Resizes the given column based on the pref width of all items contained in this column. This can be potentially
```

Comment:
I suggest two changes to documentation :
1. pref --> preferred
2. "in this column" --> "in it"

"Resizes the given column based on the preferred width of all items contained in it"